makefile

```diff
@@ -92,5 +93,14 @@ tmp/input/mondo.sssom.tsv: | tmp/input/
 	wget http://purl.obolibrary.org/obo/mondo/mappings/mondo.sssom.tsv -O $@
 
 # creates more than just this file; that goal creates multiple files
-output/medgen_terms_mapping_status.tsv output/obsoleted_medgen_terms_in_mondo.txt: | output/
+output/medgen_terms_mapping_status.tsv output/obsoleted_medgen_terms_in_mondo.txt: tmp/input/mondo.sssom.tsv | output/
 	python src/mondo_mapping_status.py
+
+# ----------------------------------------
+# Robot templates
+# ----------------------------------------
+ftp.ncbi.nlm.nih.gov/pub/meedgen/MedGenIDMappings.txt: ftp.ncbi.nlm.nih.gov
+
+# todo: Ideally I wanted this done at the end of the ingest, permuting from medgen.sssom.tsv, but there were some problems with that file
+ROBOT_addMedGen_fromIngest.tsv: ftp.ncbi.nlm.nih.gov/pub/meedgen/MedGenIDMappings.txt
+	python src/mondo_robot_template.py -i $< -o $@
```

Comment:
## Ad-hoc scripting from `MedGenIDMappings.txt`
@matentzn I had wanted to (a) create this robot template using Chris' original ingest, rather than this (b) ad-hoc method I'm using, which goes directly from `MedGenIDMappings.txt`. For 'a', my input would have been `medgen.sssom.tsv`. However, there were several problems I noticed with that file. The problems were confusing and not obvious how to solve; fixing would not have been worth the time since we may not use this ingest. So I went with your initial suggestion of 'b'.

Reply:
Done. Robot template is now in [the release](https://github.com/monarch-initiative/medgen/releases).